modules/codemod/lib/v8/index.ts

```diff
@@ -1,10 +1,13 @@
 import {Transform} from 'jscodeshift';
 
+import deprecateDrawer from './deprecateDrawer';
```

Comment:
NIT: we should rename these to `softDeprecateDrawer`

Reply:
agree!